### PR TITLE
Clean up and simplify quickstart

### LINKED ISSUE #1186: Quickstart split
Quickstart should be split for the following use cases:

- [ ] Build a CCF App
- [ ] Contribute to CCF
- [ ] Deploy a CCF App

Since we have different playbooks and containers for these scenarios now. The README ought to direct users at the relevant quickstart.

### PR DESCRIPTION
**Pull request by achamayou**
fix #1186 

Can't quite go all the way without #959, but I think the key bits here are highlighting the availability of containers, and making clear they are the easier/preferred route for users trying to build an app, as opposed to setting up an ACC VM.

### CODE REVIEW

File: doc/developers/build_app.rst [hunk 1]
```diff
@@ -73,3 +73,18 @@ Additionally, the `cchost` binary must be told that the enclave type is debug:
...
 
     $ cchost --enclave-file liblua_generic.enclave.so.debuggable --enclave-type debug [args]
+
+Build Container
+---------------
+
+With every release of CCF, a base build container is provided to facilitate reproducible builds and continuous integration.
+It contains everything needed to build and test CCF applications.
+
+.. literalinclude:: ../../docker/app_ci
```
Comment (jumaffre) [inline doc/developers/build_app.rst:83]: Is there much value including the `Dockerfile` here?

File: doc/operators/container.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Runtime Container
+=================
+
+With every release, a sample runtime base container is provided. It contains the ``cchost`` executable and its dependencies,
```
Comment (ashamis) [inline doc/operators/container.rst:4]: we should probably say for "minor release", i.e. we might provide a containers for release candidates or for some minor release (depending on need)

File: doc/developers/build_app.rst (hunk 1 above)
Comment (achamayou) [reply doc/developers/build_app.rst:83]: I think it’s useful to have a reference to it if you want to tweak it, I’ll see if I can change it to a link somehow.

File: doc/operators/container.rst (hunk 2 above)
Comment (achamayou) [reply doc/operators/container.rst:4]: I’m not sure I understand your point. I would think that we provide containers for anything that is on the Release page of GitHub, regardless of how minor it is?

File: doc/developers/build_app.rst (hunk 1 above)
Comment (achamayou) [reply doc/developers/build_app.rst:83]: Can't find a nice way to do that. It's easy for Python code, but not for arbitrary files.

File: doc/quickstart/index.rst [hunk 3]
```diff
@@ -1,31 +1,113 @@
...
+Container
+---------
 
-The install directory contains (under ``getting_started/``) the Azure and Ansible scripts required to setup the CCF environment. See :ref:`how to setup the CCF environment here <quickstart/requirements:Azure Confidential Compute>`.
+The quickest way to get started building CCF applications is to use the :ref:`developers/build_app:Build Container`.
 
-.. note::
+.. code-block:: bash
+
+    sudo docker run -it ccfciteam/ccf-app-ci:0.11
```
Comment (eddyashton) [inline doc/quickstart/index.rst:11]: Should we say `:latest` for docker versions, rather than keeping these docs in-sync manually?

Comment (achamayou) [reply doc/quickstart/index.rst:11]: We don't have a latest (yet), but that is an excellent point.

File: doc/operators/container.rst (hunk 2 above)
Comment (achamayou) [reply doc/operators/container.rst:4]: @ashamis ?

File: doc/quickstart/index.rst (hunk 3 above)
Comment (achamayou) [reply doc/quickstart/index.rst]: Done.

File: doc/operators/container.rst (hunk 2 above)
Comment (ashamis) [reply doc/operators/container.rst:4]: sure, just wanted to make sure this was part of the plan

**Pull request merged**